src/view/payment-request.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { StyleSheet, View } from 'react-native';
+import { observer } from 'mobx-react';
+import PropTypes from 'prop-types';
+import Background from '../component/background';
+import { NamedField } from '../component/field';
+import { Header, Title } from '../component/header';
+import Text from '../component/text';
+import { BackButton, CopyButton, SmallButton } from '../component/button';
+import { BalanceLabel } from '../component/label';
+import Card from '../component/card';
+import QRCode from '../component/qrcode';
+import { colors } from '../component/style';
+
+const styles = StyleSheet.create({
+  header: {
+    minWidth: 500,
+    width: '100%',
+    alignSelf: 'center',
+  },
+  content: {
+    alignItems: 'stretch',
+    justifyContent: 'flex-start',
+    overflow: 'auto',
```

Comment:
This won't work in react-native unfortunately https://stackoverflow.com/questions/39722439/react-native-overflow-and-scroll?utm_medium=organic&utm_source=google_rich_qa&utm_campaign=google_rich_qa

I've made `MainContent` scrollable via `ScrollView` in https://github.com/lightninglabs/lightning-app/commit/88354ab6345740fd25fda049a34eec2f77a392dd. Looks like we'll have to wrap our `Card` in `MainContent` after all. Here's an example of how to use it: https://github.com/lightninglabs/lightning-app/blob/5379750379741157be31d8a2cf53f3dbf53f5f21/stories/layout.js#L37-L49